src/QueryBuilder.js

```diff
@@ -1566,6 +1570,229 @@ QueryBuilder.prototype.returning = queryMethod('returning');
 QueryBuilder.prototype.truncate = queryMethod('truncate');
 
 /**
+ * Json query APIs
+ */
+
+/**
+ * Json field filter.
+ *
+ * Supports having field expression in both sides of operand, but right hand value cannot
+ * be any other value, but only string reference to other column/json filed, json array
+ * or json object.
+ *
+ * Converts left and right hand values to PostgreSQL acceptable format and add user chosen
+ * operator between left and right hand expressions.
+ *
+ * @param fieldExpression {String} Reference to column / jsonField.
+ * @param jsonObjectOrFieldExpression {Object|Array|String} Reference to column / jsonField or json object.
+ * @returns {QueryBuilder}
+ */
+QueryBuilder.prototype.whereJsonObject = function (fieldExpression, operator, jsonObjectOrFieldExpression) {
+  var fieldReference = parseFieldExpression(fieldExpression);
+  if (_.isString(jsonObjectOrFieldExpression)) {
+    var rightHandReference = parseFieldExpression(jsonObjectOrFieldExpression);
+    var refRefQuery = ["(", fieldReference, ")::jsonb ", operator, " (", rightHandReference, ")::jsonb"].join("");
+    return this.whereRaw(refRefQuery);
```

Comment:
Isn't here a possibility for an SQL injection using the operator? ` > 10; DROP TABLE "Person"; --` You should check the operator against supported ones.